### ISSUE TITLE
in-chat-view: Notification badge for messages in other conversations

### ISSUE THREAD

**Issue by wiswedel** [nextcloud/talk-android#795]
As a chat participant while chatting in one conversation, I want to be see that I have unread messages in other conversations without actively having to look for them by going back to the conversations list.
Currently, when I miss or don't immediately react on a system notiification, I won't know that there are notifications from other chats.

**Comment by mario** [comment]
System notification persists until you dismiss it though?

**Comment by wiswedel** [comment]
This is about being active in several chats at the same time. NC should show that there's something going on in other chats. Persisting system notifications would only pile up and be swept away eventually.

**Comment by mario** [comment]
Yes, they're swept away when you do it. Also, to be active in several chats at the same time you would need to have different clients connected as one client cannot be in more than one room.

I guess we could show a small dot next to the back button if there are active system notifications for unread messages.

@nickvergessen @Ivansss though?

**Comment by wiswedel** [comment]
> Also, to be active in several chats at the same time you would need to have different clients connected as one client cannot be in more than one room.

"Active" is to be understood as "actively involved" (demanded person, switching from conversation to conversation all the time).

> I guess we could show a small dot next to the back button if there are active system notifications for unread messages.

That'd be great.

**Comment by nickvergessen** [comment]
What do other messengers do?

**Comment by Ivansss** [comment]
> I guess we could show a small dot next to the back button if there are active system notifications for unread messages.

Yeah. I guess we could show the unread messages/conversations number next to the back button.
Something like:

< (2)

**Comment by mario** [comment]
@nickvergessen at least signal and viber don't do anything.

**Comment by wiswedel** [comment]
from WhatsApp:

![image](https://user-images.githubusercontent.com/1804221/80374992-d4cf0e00-8897-11ea-8c24-7797f9aa08aa.png)

**Comment by wiswedel** [comment]
Telegram

![image](https://user-images.githubusercontent.com/1804221/80375121-0e077e00-8898-11ea-8111-f64d76347a82.png)

**Comment by mario** [comment]
Ok, so that's number of conversations where you're notified. We can do this I guess.

**Comment by wiswedel** [comment]
related: https://github.com/nextcloud/talk-ios/issues/301

**Comment by nickvergessen** [comment]
> Ok, so that's number of conversations where you're notified. We can do this I guess.

sounds good
